GeeksCoreLibrary/Modules/PostalServices/PostNL/Controllers/PostNLController.cs

```diff
@@ -0,0 +1,231 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using GeeksCoreLibrary.Core.Extensions;
+using GeeksCoreLibrary.Core.Interfaces;
+using GeeksCoreLibrary.Core.Models;
+using GeeksCoreLibrary.Modules.Communication.Interfaces;
+using GeeksCoreLibrary.Modules.GclReplacements.Interfaces;
+using GeeksCoreLibrary.Modules.PostalServices.PostNL.Interfaces;
+using GeeksCoreLibrary.Modules.PostalServices.PostNL.Models;
+using GeeksCoreLibrary.Modules.PostalServices.PostNL.Services;
+
+namespace GeeksCoreLibrary.Modules.PostalServices.PostNL.Controllers
+{
+    [Route("api/post-nl")]
```

Comment:
Graag niet `/api` gebruiken, want de GCL is geen API. Kun je hier `/postal-services/post-nl` van maken a.u.b.?